### ISSUE TITLE
[v8.x] Lint failures in CI due to upstream preact module export change

### ISSUE THREAD

**Issue by edmorley** [neutrinojs/neutrino#1078]
Travis tests have started failing on the `release/v8` branch, with:

```
create-project › test › cli_test › @neutrinojs/preact + @neutrinojs/airbnb Failed to lint project:
$ neutrino lint
error: Missing file extension "mjs" for "preact" (import/extensions) at src/App.jsx:1:27:
> 1 | import { Component } from 'preact';
    |                           ^
  2 | import './App.css';
  3 | 
  4 | export default class App extends Component {
```
(eg https://travis-ci.org/neutrinojs/neutrino/jobs/425302165#L860)

This appear to be due to:
https://github.com/developit/preact/pull/1166

And the fact that `eslint-config-airbnb-base@^16` (which is what is used by Neutrino 8), doesn't have .`mjs` in the extensions list for the `import/extensions` rule:
https://github.com/airbnb/javascript/blob/cf9e17efee72121b17507b030c04f9b11c521a9b/packages/eslint-config-airbnb-base/rules/imports.js#L132-L137

...whereas `eslint-config-airbnb-base@^17` (which is what is used by Neutrino `master`), does include `.mjs` in that rule:
https://github.com/airbnb/javascript/blob/02b4eea3452b0ae6da666b01f9da23f26148f5b6/packages/eslint-config-airbnb-base/rules/imports.js#L19-L23

**Comment by edmorley** [comment]
Wontfix:
https://github.com/neutrinojs/neutrino/pull/1109#issuecomment-422579731

**Comment by disbelief** [comment]
For people who might be blocked by this (since `neutrino build` bails out when linting fails), here's what I added to `.neutrinorc.js` to fix it in my neutrino-created app:

```javascript
module.exports = {
  use: [
    [
      '@neutrinojs/airbnb',
      {
        eslint: {
          rules: {
            "import/extensions": [
              "error",
              "always",
              {
                "js": "never",
                "jsx": "never",
                "mjs": "never"
              }
            ]
          }
        }
      }
    ],
    [
      '@neutrinojs/preact',
      {
        html: {
          title: 'neutrinopreact'
        }
      }
    ]
]
```
